common/github/request_service.go

```diff
@@ -9,10 +9,13 @@ import (
 )
 
 // GitHubGophrPackageOrgName is the  Github organization name for all versioned packages
-// GitHubBaseAPIURL is the base Github API
 const (
 	GitHubGophrPackageOrgName = "gophr-packages"
-	GitHubBaseAPIURL          = "https://api.github.com"
+)
+
+// GitHubBaseAPIURL is the base Github API
+const (
+	GitHubBaseAPIURL = "https://api.github.com"
```

Comment:
My linter was complaining about them needing export comments previously but when the comments are on top of each other they look really botched on GoDoc,  so I separated them.